chatbot/admin.py

```
@@ -0,0 +1,303 @@
+        }),
+        ('محتوا', {
+            'fields': ('summary', 'tags'),
+            'classes': ('collapse',)
+        }),
+        ('داده‌ها', {
+            'fields': ('metadata',),
+            'classes': ('collapse',)
+        }),
+    )
+    
+    def session_user(self, obj):
```

Ruff (0.12.2)

Missing return type annotation for public function `session_user`

(ANN201)